app/travis_scripts/build-docker-env.sh

```diff
@@ -24,12 +24,17 @@ RETRIES=30;
 RETRY_DELAY=5s;
 
 # wait for E&E to come up
-waitFor $RETRIES $RETRY_DELAY checkHTTP "engine-and-editor" 401 http://localhost/api/v1/users/me;
+waitFor $RETRIES $RETRY_DELAY checkHTTP "engine-and-editor" 302 http://localhost/api/v1/users/me;
```

Comment:
Hmm, I think backend returns 302 (=redirect) because we are not sending an authentication token with the request. Is there some other endpoint which does not require authentication we could use?

Reply:
yeah not sure, this is how the other envs check if it's up so 🤷‍♀The actual code probably doesn't matter too much, just so long as it's not `5xx`.